tests/test_generate.py

```diff
@@ -63,6 +97,27 @@ def init_test():
     return test_bucket, storage_client
 
 
+def cleanup_bucket(storage_client, test_bucket):
```

Comment:
Nit (verb instead of adjective):
```suggestion
def clean_up_bucket(storage_client, test_bucket):
```